pkg/vadmin/add_node_vc.go

```diff
@@ -66,6 +66,9 @@ func (v *VClusterOps) genAddNodeOptions(s *addnode.Parms, certs *HTTPSCerts) vop
 	*opts.HonorUserInput = true
 	*opts.ForceRemoval = true
 	*opts.SkipRebalanceShards = true
+	opts.ExpectedNodeNames = s.ExpectedNodeNames
+	v.Log.Info(fmt.Sprintf("Existing nodes in catalog: %+v", opts.ExpectedNodeNames))
+	v.Log.Info(fmt.Sprintf("New hosts to add: %+v", opts.NewHosts))
```

Comment:
We shouldn't be building up a formatted string. It's a structured log. So you would typically include a static string, then pass in key/value parms for the things you are interested in.
```suggestion
	v.Log.Info("Nodes in request", "ExpectedNodeNames", opts.ExpectedNodeNames, "NewHosts", opts.NewHosts)
```